js/task-tracker.js

```diff
@@ -102,6 +120,26 @@ class Task {
         `;
         return html;
     }
+
+
+    static formatDate(date) {
+
+        let hh = date.getHours();
+        if (hh < 10) hh = '0' + hh;
```

Comment:
Please write new code from new line and wrap it in curly braces, use string interpolation

Reply:
Thanks, now it looks like this
```suggestion
        if (hh < 10) {
            hh = `0${hh}`;
        }
```